community/libmustache/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Renoir Boulanger <hello@renoirboulanger.com>
+# Maintainer: John Boehr <jbboehr@gmail.com>
```

Comment:
Do they agree with maintaining the package? Note that the maintainer tag is about who maintains the APKBUILD, not the upstream package.